docker-compose.yml

```diff
@@ -42,12 +53,15 @@ services:
     depends_on:
       postgres:
         condition: service_healthy
+      test-postgres:
+        condition: service_started
```

Comment:
Does this mean it's going to run two instances of the db?

Reply:
Yes, it does. There's the option of not adding this to the depends-on and letting people manually invoke it if we don't want that.

Reply:
Well, Docker Compose is intended to only be used for development purposes, right? So why don't we just always disable durability? I don't think anyone would cry over development data being lost in the rare case of power loss or a crash.

Reply:
That's a fair point, I don't see a good reason not to.